src/main/java/hu/oe/nik/szfmv/automatedcar/model/deserializer/Deserializer.java

```diff
@@ -0,0 +1,96 @@
+package hu.oe.nik.szfmv.automatedcar.model.deserializer;
+
+import com.google.gson.*;
+import com.google.gson.reflect.TypeToken;
+import hu.oe.nik.szfmv.automatedcar.model.WorldObject;
+
+import java.io.*;
+import java.lang.reflect.Type;
+import java.util.*;
+
+public class Deserializer {
+    private static void TestFile(String fileName) throws IllegalArgumentException {
+        if (!fileName.contains("json")) {
+            throw new IllegalArgumentException("Provided file's extension is not .json! \n File:" + fileName);
+        }
+    }
+
+    private static String ReadFile(String fileName) {
```

Comment:
Ez egy teljesen fölösleges munka volt.

Reply:
Ez csak be van mergelve a team2-től. Mi csak a saját elemeinket csináltuk meg, de így jelenik meg.

Reply:
Team 2-től mergelve.

Reply:
Rájöttem később, hogy volt egy csapatközi merge, ez a PR-ből alapból nekem nem volt egyértelmű.